### ISSUE TITLE
[docker] automatically release docker image

### ISSUE THREAD

**Issue by bukepo** [openthread/ot-br-posix#274]
**Is your feature request related to a problem? Please describe.**
docker image is out of date, and nobody remembered to update it.

**Describe the solution you'd like**
Use Travis to periodically release new docker image for otbr.

**Describe alternatives you've considered**
Not sure how docker hub will authenticate. Maybe the job should be created in docker hub.

**Additional context**
None.

**Comment by gjc13** [comment]
Reference: https://docs.docker.com/docker-hub/builds/

**Comment by gjc13** [comment]
@jwhui We need to add a bot account to OpenThead team on github and docker hub so that docker hub can be linked to the code repository.

**Comment by jwhui** [comment]
@gjc13, I enabled autobuild on docker hub for this repo. However, the builds are currently failing:
```
/bin/mkdir -p '/app/borderrouter/._stage/usr/include/CppUTest'
/usr/bin/install -c -m 644 include/CppUTest/CommandLineArguments.h include/CppUTest/CommandLineTestRunner.h include/CppUTest/CppUTestConfig.h include/CppUTest/JUnitTestOutput.h include/CppUTest/MemoryLeakDetector.h include/CppUTest/MemoryLeakDetectorMallocMacros.h include/CppUTest/MemoryLeakDetectorNewMacros.h include/CppUTest/MemoryLeakWarningPlugin.h include/CppUTest/PlatformSpecificFunctions.h include/CppUTest/PlatformSpecificFunctions_c.h include/CppUTest/SimpleString.h include/CppUTest/SimpleMutex.h include/CppUTest/StandardCLibrary.h include/CppUTest/TeamCityTestOutput.h include/CppUTest/TestFailure.h include/CppUTest/TestFilter.h include/CppUTest/TestHarness.h include/CppUTest/TestHarness_c.h include/CppUTest/TestMemoryAllocator.h include/CppUTest/TestOutput.h include/CppUTest/TestPlugin.h include/CppUTest/TestRegistry.h include/CppUTest/TestResult.h include/CppUTest/TestTestingFixture.h include/CppUTest/Utest.h include/CppUTest/UtestMacros.h generated/CppUTestGeneratedConfig.h '/app/borderrouter/._stage/usr/include/CppUTest'
/bin/mkdir -p '/app/borderrouter/._stage/usr/include/CppUTestExt'
/usr/bin/install -c -m 644 include/CppUTestExt/GMock.h include/CppUTestExt/GTest.h include/CppUTestExt/GTestConvertor.h include/CppUTestExt/IEEE754ExceptionsPlugin.h include/CppUTestExt/MemoryReportAllocator.h include/CppUTestExt/MemoryReporterPlugin.h include/CppUTestExt/MemoryReportFormatter.h include/CppUTestExt/MockActualCall.h include/CppUTestExt/MockCheckedActualCall.h include/CppUTestExt/MockCheckedExpectedCall.h include/CppUTestExt/MockExpectedCall.h include/CppUTestExt/MockExpectedCallsList.h include/CppUTestExt/MockFailure.h include/CppUTestExt/MockNamedValue.h include/CppUTestExt/MockSupport.h include/CppUTestExt/MockSupportPlugin.h include/CppUTestExt/MockSupport_c.h include/CppUTestExt/OrderedTest.h '/app/borderrouter/._stage/usr/include/CppUTestExt'
/bin/mkdir -p '/app/borderrouter/._stage/usr/lib/pkgconfig'
/usr/bin/install -c -m 644 cpputest.pc '/app/borderrouter/._stage/usr/lib/pkgconfig'
make[1]: Leaving directory '/tmp/build-1556303228/cpputest-3.8'
fatal: not a git repository: /src/bq8jrkrgqvg4bgjnsqff29u/.git/modules/third_party/openthread/repo
Unable to find current revision in submodule path 'third_party/openthread/repo'
Removing intermediate container 2cf2181e886a
The command '/bin/sh -c cd /app/borderrouter && NAT64=1 ./script/bootstrap' returned a non-zero code: 1
```
Thoughts?

**Comment by bukepo** [comment]
Is submodule initialized? @jwhui

**Comment by jwhui** [comment]
Shouldn’t that be covered in script/bootstrap?

**Comment by bukepo** [comment]
I'm sorry, but I don't have docker access to debug this issue. could you grant me the access? my account is bukepo same as github account.

**Comment by bukepo** [comment]
Shall we create periodic tags on docker hub? Or just use the latest?

**Comment by jwhui** [comment]
@bukepo, are you suggesting to apply release names?

**Comment by jwhui** [comment]
Resolved by #518
